src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/NoSentenceTest.kt

```diff
@@ -21,7 +21,6 @@ class NoSentenceTest : MockedEndpointsTestBase() {
     setUpNoSentence(crn)
     setupRegistrations(registrationsResponse(), crn)
     restOfSetupWithMaleOffenderNoSevereNeeds(crn)
-    tierUpdateWillSucceed(crn, "A0")
```

Comment:
this has an assertion in it, I don't think this test has any assertions any more

Reply:
good spot, I thought I did them all, I've added the same repo assertion as the rest for now.